toolkit/components/uniffi-bindgen-gecko-js/fixtures/Arithmetic.jsm

```diff
@@ -201,7 +215,13 @@ class FfiConverterArrayBuffer {
     }
 }
 
-class FfiConverterU64 {
+class FfiConverterU64 extends FfiConverter {
+    static checkType(name, value) {
+        super.checkType(name, value);
+        if (!Number.isSafeInteger(value)) {
+            throw TypeError(`${name} exceeds the safe integer bounds (${value})`);
```

Comment:
nano-nit - maybe `${name} is outside safe integer bounds: (${value})` (ie, colon before the value, and avoid "exceeds" because negative numbers can trigger this.